lib/Dialect/TorchConversion/Transforms/VerifyTosaLinalgBackendContract.cpp

```diff
@@ -84,9 +86,10 @@ class VerifyTosaLinalgBackendContractPass
     target.addDynamicallyLegalDialect<affine::AffineDialect>(opHasLegalTypes);
     target.addDynamicallyLegalDialect<cf::ControlFlowDialect>(opHasLegalTypes);
     target.addDynamicallyLegalDialect<scf::SCFDialect>(opHasLegalTypes);
-    target.addDynamicallyLegalDialect<ml_program::MLProgramDialect>(
-        opHasLegalTypes);
-
+    target.addDynamicallyLegalDialect<ml_program::MLProgramDialect>(opHasLegalTypes);
+    target.addLegalDialect<memref::MemRefDialect>();
+    target.addLegalDialect<bufferization::BufferizationDialect>();    
+    
```

Comment:
Use `addDynamicallyLegalDialect` to be consistent with the other dialect checks.

Reply:
Fixed